utils/ui.js

```diff
@@ -65,5 +65,7 @@ export const etherscanLink = (string, network = 'mainnet') => {
 };
 
 export const oasisLink = (string, network) => {
-  return `http://${network === 'kovan' ? 'staging.' : ''}oasis.app${string}`;
+  return `http://oasis.app${string}${
```

Comment:
@levity since `staging.oasis.app` is down, I removed links to it. Is there another url we should link to for kovan testing or do you think linking directly to `oasis.app` ok?

Reply:
yeah this is fine for now